dataplane/ebpf/src/egress/tcp.rs

```diff
@@ -68,6 +71,39 @@ pub fn handle_tcp_egress(ctx: TcContext) -> Result<i32, i64> {
     unsafe { (*tcp_hdr).check = 0 };
 
     // TODO: connection tracking cleanup https://github.com/kubernetes-sigs/blixt/issues/85
```

Comment:
depends on what the scope of the issue is, since its a bit vague. i'll add this back for now, we can revisit it later.

Reply:
Well this is me suggesting to _remove_ one, not me asking about one that was removed. If there's still more to do it might be worth creating a follow up issue to enumerate those things and update the TODO(s) accordingly?